src/gauge/builderRewardsClaimed.ts

```diff
@@ -0,0 +1,25 @@
+import { BuilderRewardsClaimed as BuilderRewardsClaimedEvent } from "../../generated/templates/GaugeRootstockCollective/GaugeRootstockCollective";
+import { Builder, BuilderRewardsClaimed } from "../../generated/schema";
+import { BigInt } from "@graphprotocol/graph-ts";
+
+export function handleBuilderRewardsClaimed(
+  event: BuilderRewardsClaimedEvent
+): void {
+  const builder = Builder.load(event.params.builder_);
+  if (builder == null) return;
+
+  const rewardsClaimedId = event.params.builder_.concat(
+    event.params.rewardToken_
+  );
+  let rewardsClaimed = BuilderRewardsClaimed.load(rewardsClaimedId);
+  if (rewardsClaimed == null) {
+    rewardsClaimed = new BuilderRewardsClaimed(rewardsClaimedId);
+    rewardsClaimed.amount_ = BigInt.zero();
```

Comment:
I don't think we need to set the amount here

Reply:
We need it since we use it 4 lines below, so it should be initialized.